ext/snmp/tests/snmp_set_string_output_format.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Function snmp_set_string_output_format
+--CREDITS--
+Larry Adams
+--SKIPIF--
+<?php
+require_once(__DIR__.'/skipif.inc');
+if (!function_exists('snmp_set_string_output_format')) die('skip This function is only available if using NET_SNMP');
+?>
+--FILE--
+<?php
+require_once(__DIR__.'/snmp_include.inc');
+
+echo "Checking error handling\n";
+try {
+    var_dump(snmp_set_string_output_format(123));
+} catch (\ValueError $e) {
+    echo $e->getMessage() . \PHP_EOL;
+}
+
+echo "Checking working\n";
+var_dump(snmp_set_string_output_format(SNMP_STRING_OUTPUT_GUESS));
+var_dump(snmp_set_string_output_format(SNMP_STRING_OUTPUT_ASCII));
+var_dump(snmp_set_string_output_format(SNMP_STRING_OUTPUT_HEX));
```

Comment:
We should also fetch something here to see that the output format actually has some effect.

Reply:
Yea, the trick is, it'll be different on each system.  I'll guess we can request that the sysLocation be set in the snmpd.conf and that way it'll always succeed.  Add that to the test configuration.